### PR TITLE
fix: avoid saving wrong identity id when api key is incorrect

### PR DESCRIPTION
**Pull request by xitronix**
Closes https://github.com/arcxmoney/analytics-sdk/issues/91

### CODE REVIEW

File: src/ArcxAnalyticsSdk.ts [hunk 1]
```diff
@@ -391,6 +391,16 @@ export class ArcxAnalyticsSdk {
   /** Initialises the Analytics SDK with desired configuration. */
   static async init(apiKey: string, config?: Partial<SdkConfig>): Promise<ArcxAnalyticsSdk> {
     const sdkConfig = { ...DEFAULT_SDK_CONFIG, ...config }
+    /*
+      TODO: Remove this if statement aproximately in February.
+      Clean local storage in case if users with previous versions saved wrong indentity_id when ARCX_ANALYTIC_API_KEY was incorrect
```
Comment (kQuintal) [inline src/ArcxAnalyticsSdk.ts:396]: Why Feb? Also I would add the year to this (e.g. Feb 2023)

Also, if its a TODO, I would add a github issue and make it priority so we don't forget.

File: src/ArcxAnalyticsSdk.ts [hunk 2]
```diff
@@ -391,6 +391,16 @@ export class ArcxAnalyticsSdk {
...
+    */
+    if (
+      window.localStorage.getItem(IDENTITY_KEY)?.length !== 64 &&
+      window.localStorage.getItem(IDENTITY_KEY)?.length !== 0
```
Comment (kQuintal) [inline src/ArcxAnalyticsSdk.ts:400]: Can we remove second statement? If IDENTITY_KEY is found without being 64 characters, then doesn't that mean it shouldn't exist, so we always remove it? 

I guess I can't see a condition where `getItem(..)` is defined and not zero? But maybe I'm missing something

**Review by kQuintal — CHANGES_REQUESTED** [review]

File: src/ArcxAnalyticsSdk.ts (hunk 1 above)
Comment (xitronix) [reply src/ArcxAnalyticsSdk.ts:396]: https://github.com/arcxmoney/analytics-sdk/issues/96

File: src/ArcxAnalyticsSdk.ts (hunk 2 above)
Comment (xitronix) [reply src/ArcxAnalyticsSdk.ts:400]: I think you have the point, so I'll remove it.

**Review by kQuintal — APPROVED** [review]
🚀

**Pull request merged**